data_collection/web/javascript_tracking_client/api.rst

```diff
@@ -2407,12 +2407,11 @@ JavaScript Tracking Client configuration
 .. function:: setApiUrl(url)
```

Comment:
Shoudn't it be `setAPIUrl`? with API in capital?

Reply:
I'm just describing current public methods. We can clean them up later using deprecation process.

Reply:
But the source code has `API`, or am I seeing things?
![image](https://user-images.githubusercontent.com/16927151/165063151-d290845d-e31f-4ea2-aee6-613b90c1897e.png)